provision/ansible/playbooks/install-sync-gateway-package.yml

```diff
@@ -36,7 +36,8 @@
 
   tasks:
   - include: tasks/flush-server-buckets.yml
-
+    when: skip_bucketflush == "false"
+    
```

Comment:
> Does a python False get correctly converted to "false" here? Just double checking.

It seemed to in my testing.  To be honest I was confused on how to do this comparison .. I'll try to switch to compare against False (boolean) and see what happens